syft/exceptions.py

```diff
@@ -273,11 +273,11 @@ def __init__(self, obj_id, worker):
             + " which does not exist!!! "
         )
         message += (
-            "Use .send() and .get() on all your tensors to make sure they're"
+            "Use .send() and .get() on all your tensors to make sure they're "
```

Comment:
Not related to the issue. When a string is split on multiple lines like this the printed resulted string would be all the chunks concatenated - need to add space between words :)